src/MiFare/Classic/MiFareStandardCardReaderBase.cs

```diff
@@ -22,9 +22,9 @@ abstract class MiFareStandardCardReaderBase : ICardReader, IDisposable
        
         private Dictionary<SectorKey, byte[]> keyMap = new Dictionary<SectorKey, byte[]>();
 
-        private byte nextKeySlot;
+        private static byte nextKeySlot;
```

Comment:
These can't be static since it wouldn't apply to a different card if placed under the reader.

Reply:
Ah sorry didn't think about this case. Maybe instead it would make sense to just remove the IDisposable interface? Or use as static and clear when card's removed?